config/201-rolebinding.yaml

```diff
@@ -102,3 +102,39 @@ roleRef:
   apiGroup: rbac.authorization.k8s.io
   kind: Role
   name: tekton-pipelines-info
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: tekton-events-controller
+  namespace: tekton-pipelines
+  labels:
+    app.kubernetes.io/component: events
+    app.kubernetes.io/instance: default
+    app.kubernetes.io/part-of: tekton-pipelines
+subjects:
+  - kind: ServiceAccount
+    name: tekton-events-controller
+    namespace: tekton-pipelines
+roleRef:
+  kind: Role
+  name: tekton-pipelines-controller
```

Comment:
These are tied to the controller config options - not sure if you want the events controller to share the same config options as the pipeline controller or if you want to have it's own set of configs for flexibility.

Reply:
My plan was for this PR to share the same configs, to keep the change as small as possible for users as well.
Once `TaskRun` and `PipelineRun` events are available in the new controller as well, I would start adding dedicated configurations.

Reply:
👍 Fine with me, just wanted to make sure it was intentional 🙂